license-classifications.yml

```
@@ -1712,15 +1716,17 @@ categorizations:
 
   - id: "CC-BY-NC-SA-4.0"
     categories:
-      - "source-available"
+      - "copyleft-strong"
```

```suggestion
      - "free-restricted"
```
As all CC-licenses with the NC-addition are not open source, we do not apply the catergory "copyleft-strong" to these. Insteast the category "free-restricted" would seem the most appropriate, as long as the "property:noncommercial" is also in place.